tests/e2e/test_rosacli_cluster.go

```diff
@@ -1018,6 +1018,19 @@ var _ = Describe("Classic cluster creation validation",
 				Expect(errorOutput.String()).To(ContainSubstring("etcd encryption cannot be disabled on clusters with FIPS mode"))
 			})
 
+		It("validate use-local-credentials won't work with sts - [id:65900]",
```

Comment:
Sorry it took a while to get to your feedback. But, I believe this test doesn't use the profiles at all, so since I specify the `--sts` flag it should always be an STS cluster